rpc/src/endpoint/genesis_chunked.rs

```diff
@@ -0,0 +1,52 @@
+//! `/genesis_chunked` endpoint JSON-RPC wrapper
+
+use alloc::{string::String, vec::Vec};
+use serde::{Deserialize, Serialize};
+use tendermint_proto::serializers;
+
+use crate::{dialect::Dialect, request::RequestMessage};
+
+/// Get the genesis state in multiple chunks for the current chain
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct Request {
+    pub chunk: Option<String>,
+}
+
+impl Request {
+    pub fn new(chunk: Option<String>) -> Self {
+        Self { chunk }
+    }
+}
+
+impl RequestMessage for Request {
+    fn method(&self) -> crate::Method {
+        crate::Method::GenesisChunked
+    }
+}
+
+impl<S> crate::Request<S> for Request
+where
+    S: Dialect,
+{
+    type Response = Response;
+}
+
+impl<S> crate::SimpleRequest<S> for Request
+where
+    S: Dialect,
+{
+    type Output = Response;
+}
+
+/// Block responses
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Response {
+    #[serde(with = "serializers::from_str")]
+    pub chunk: u64,
+    #[serde(with = "serializers::from_str")]
+    pub total: u64,
```

Comment:
These two are also supposed to be integers as far as I can tell.

Reply:
I seem to need to convert it or else I'm getting:

```
query to /genesis_chunked failed with error: 
   0: serde parse error
   1: invalid type: string "0", expected i32 at line 1 column 82
```

Reply:
Alright, my bad, was mislead by the docs.